src/main/kotlin/racingcar/domain/car/Cars.kt

```diff
@@ -0,0 +1,17 @@
+package racingcar.domain.car
+
+import racingcar.domain.GasStation
+
+class Cars(
+    val values: List<Car>
+) {
+    fun race(gasStation: GasStation) = values.forEach { car ->
+        car.fillGasAndForward(gasStation.getRandomGas())
+    }
+
+    fun getWinnerNames(): List<String> {
+        val winnerDistance = values.maxOf { it.distance }
+        return values.filter { it.distance == winnerDistance }
+            .map { it.getName() }
```

Comment:
asSequence()를 활용하면, 매번 컬렉션을 만들어내지 않고 Java의 Stream처럼 사용이 가능합니다 :)

 ```suggestion
        return values.asSequence()
            .filter { it.distance == winnerDistance }
            .map { it.getName() }
            .toList()
```